include/drjit-core/optix.h

```diff
@@ -68,6 +68,15 @@ jit_optix_configure_pipeline(const OptixPipelineCompileOptions *pco,
 extern JIT_EXPORT uint32_t
 jit_optix_configure_sbt(const OptixShaderBindingTable *sbt, uint32_t pipeline);
 
+/**
+ * \brief  Update existing OptiX Shader Binding Table data
+ *
+ * This function updates the Shader Binding Table data hold by the JIT
+ * variable \c index previously created using \c jit_optix_configure_sbt.
```

Comment:
Could you add a line saying what parts of the SBT updated, and in what context such an update becomes necessary?